MAUI/TabView/Swiping.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: About .NET MAUI Tab View (SfTabView) control | Syncfusion
+description: Learn here all about swiping support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.
```

Comment:
@Arunachalam3641, Use the content below,
Learn here all about the swiping support in Syncfusion .NET MAUI Tab View (SfTabView) control and more.

Reply:
Yes, I changed the content.